plugin/src/main/java/org/opensearch/ml/processor/MLInferenceIngestProcessor.java

```diff
@@ -38,14 +37,18 @@
  */
 public class MLInferenceIngestProcessor extends AbstractProcessor implements ModelExecutor {
 
+    public static final String DOT_SYMBOL = ".";
     private final InferenceProcessorAttributes inferenceProcessorAttributes;
     private final boolean ignoreMissing;
     private final boolean ignoreFailure;
     private final ScriptService scriptService;
     private static Client client;
     public static final String TYPE = "ml_inference";
     public static final String DEFAULT_OUTPUT_FIELD_NAME = "inference_results";
+    // allow to ignore a field from mapping is not present in the document, and when the outfield is not found in the
+    // prediction outcomes, return the whole prediction outcome by skipping filtering
     public static final String IGNORE_MISSING = "ignore_missing";
+    // At default, ml inference processor allows maximum 10 prediction tasks running in parallel
```

Comment:
Can user override it ? Explain this in the comment?

Reply:
added comment, can be override.